package.nls.json

```diff
@@ -1,4 +1,5 @@
 {
+  "command.activateIcons.title": "Activate VSCode Icon Theme",
```

Comment:
I would use the same solution that `Material Icons` uses: `Activate Icon Theme`. No confussion and we stick to the same convention.